.github/workflows/notify-triager.yml

```diff
@@ -68,7 +69,7 @@ jobs:
           echo "codeTriagers=$codeTriagers" >> $GITHUB_ENV
 
       - name: Add Reviewers for code files
-        if: steps.check-merge-branch.outputs.isMergeCommit == 'false' && steps.non-md-pr-changes.outputs.any_changed == 'true'
+        if: ${{ steps.check-merge-branch.outputs.isMergeCommit == 'false' && steps.changed-files.outputs.all_changed_files_count != steps.md-pr-changes.outputs.all_changed_files_count }}
```

Comment:
What if the `changed-files.outputs.all_changed_files_count` is equal to `steps.md-pr-changes.outputs.all_changed_files_count`? 
Can you please explain exactly what that check means?

Reply:
For this we are checking for non md files
If the count is not equal that means there are some non md files also which needs to be reviewed by code reviewers

Reply:
Got it. Good catch

Reply:
Though I am still curious about the ambigious behavior of this workflow. For example, examine this PR

https://github.com/asyncapi/website/pull/3245#issuecomment-2388259456

See how MD triagers are getting assigned as PR reviewers even though there was no merge commit or a MD file change

Reply:
https://github.com/asyncapi/website/actions/runs/11141478939/job/30962427072

I think this job was surpassed